config/locales/en/admin.yml

```diff
@@ -1,4 +1,6 @@
 en:
+  true_value: 'Yes'
+  false_value: 'No'
```

Comment:
I think it would be better move this keys to `shared:` in this file:

```
admin:
  (...)
  shared:
    true_value: 'Yes'
    false_value: 'No'
```

To use:
`t("admin.shared.true_value") and t("admin.shared.false_value")`

😉